packages/generator/ni_measurement_plugin_sdk_generator/client/templates/measurement_plugin_client.py.mako

```diff
@@ -138,9 +186,15 @@ class ${class_name}:
                 service_name=f"{self._service_class}.Configurations",
             )
         )
-        return v2_measurement_service_pb2.MeasureRequest(
-            configuration_parameters=serialized_configuration
-        )
+        if self._pin_map_context is None:
+            return v2_measurement_service_pb2.MeasureRequest(
+                configuration_parameters=serialized_configuration,
+            )
+        else:
+            return v2_measurement_service_pb2.MeasureRequest(
+                configuration_parameters=serialized_configuration,
+                pin_map_context=self._pin_map_context._to_grpc(),
+            )
```

Comment:
Protobuf allows setting pin_map_context=None, so you don't have to duplicate the entire constructor call.
```suggestion
        return v2_measurement_service_pb2.MeasureRequest(
            configuration_parameters=serialized_configuration,
            pin_map_context=self._pin_map_context._to_grpc() if self._pin_map_context else None,
        )
```